configuration/config-txt/overclocking.md

```diff
@@ -39,22 +39,22 @@ Overclocking and overvoltage will be disabled at runtime when the SoC reaches 85
 
 This table gives the default values for the options on various Raspberry Pi Models, all frequencies are stated in MHz.
 
-| Option       | Pi 0/W | Pi1 | Pi2 | Pi3   | Pi3A+/Pi3B+ | Pi4  |
-| ---          | :---:  | :---: | :---: | :----:  | :-----: | :----: |
-| arm_freq     | 1000   | 700 | 900 | 1200  | 1400  | 1500 |
-| core_freq    | 400    | 250 | 250 | 400   | 400   | 500/550/360 |
-| h264_freq    | 300    | 250 | 250 | 400   | 400   | 500/550/360 |
-| isp_freq     | 300    | 250 | 250 | 400   | 400   | 500/550/360 |
-| v3d_freq     | 300    | 250 | 250 | 400   | 400   | 500/550/360 |
-| hevc_freq    | N/A    | N/A | N/A | N/A   | N/A   | 500/550/360 |
-| sdram_freq   | 450    | 400 | 450 | 450   | 500   | 3200 |
-| arm_freq_min | 700    | 700 | 600 | 600   | 600   | 600  |
-| core_freq_min| 250    | 250 | 250 | 250   | 250   | 250/275 |
-| gpu_freq_min | 250    | 250 | 250 | 250   | 250   | 500  |
-| h264_freq_min|  250   | 250 | 250 | 250   | 250   | 500  |
-| isp_freq_min |  250   | 250 | 250 | 250   | 250   | 500  |
-| v3d_freq_min |  250   | 250 | 250 | 250   | 250   | 500  |
-| sdram_freq_min |400   | 400 | 400 | 400   | 400   | 400  |
+| Option       | Pi 0/W | Pi1 | Pi2 | Pi3   | Pi3A+/Pi3B+ | Pi4/CM4  | Pi 400 |
+| ---          | :---:  | :---: | :---: | :----:  | :-----: | :----: | :----: |
+| arm_freq     | 1000   | 700 | 900 | 1200  | 1400  | 1500 | 1800 | 
+| core_freq    | 400    | 250 | 250 | 400   | 400   | 500/550 | 500/550 |
+| h264_freq    | 300    | 250 | 250 | 400   | 400   | 500 | 500|
```

Comment:
I believe these should be `500/550` for all gpu clocks (after "platform: Increase gpu freqs to 550 with enable_4kp60")

Reply:
Does tvout still make a difference or is that 500 (or close to) after the recent clock improvments?

Reply:
> I believe these should be `500/550` for all gpu clocks (after "platform: Increase gpu freqs to 550 with enable_4kp60")

There's no explanation of why there are two values here (i.e. 500/550). I think I'm right in saying the first number is the standard clock, while the second is with `hdmi_enable_4kp60`. To make the table less complicated, my suggestion would be to just list 500 as the clock speed, but put a superscripted 1 after it, then below the table have `1. If hdmi_enable_4kp60=1 is set in config.txt then this clock defaults to 550MHz.`

Superscripted characters need to be inserted using HTML, since there seems to be no markdown equivalent. The HTML would be `<sup>1</sup>`.

Reply:
tvout doesn't affect core clock

Reply:
Changed to 500. There's a separate section for 4kp60 which notes that max core freq is increased to 550 MHz